cli/copr_cli/main.py

```diff
@@ -308,12 +308,15 @@ def action_build(self, args):
         :param args: argparse arguments provided by the user
         """
 
-        # Before we start uploading potentially large source RPM file, make sure
-        # that the user actually has a valid credentials.
-        self.client.base_proxy.auth_check()
```

Comment:
This removal has a small potential to cause trouble.  It actually did a real `/auth-check` call before - revealing that session expired (e.g. wiped-out redis DB because frontend restarted).

Keeping this has an obvious consequence - additional API call for each build request.

Reply:
Good catch. 
There are two commits in this PR. This issue can be easily solved by removing the second commit. We can do it that way or any other way you prefer.